packages/@angular/cli/lib/config/schema.json

```diff
@@ -473,6 +473,11 @@
               "description": "The ssl certificate used by the server.",
               "type": "string",
               "default": "ssl/server.crt"
+            },
+            "proxyConfig": {
+              "description": "Proxy configuration file.",
+              "type": "string",
+              "default": ""
```

Comment:
Can you remove the default here? It's better to not have one since sending in an empty string is different than not sending anything.